src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
Thanks @mbaudis! Is there an issue? I would strongly prefer the name to be included. I would also suggest dropping the 'ontology' prefix from all attributes for consistency with the rest of the API.

Usig the asbove example, this would look something like:

{"sourceName":"Sequence Ontology",
"sourceVersion":"release_2.5.3"
"name":"missense_variant",
"id":"http://www.sequenceontology.org/browser/release_2.5.3/term/SO:0001583"}